### DISCUSSION TITLE
How to change Network speed

### DISCUSSION THREAD

**Discussion by Muthu-Krishnan-L** [GoogleChrome/lighthouse#15080] [Q&A]
Hi all,

I would like to modify the Network speed and CPU processing power for Mobile and Web Lighthouse Performance testing. Please let me know if there is a way to do it manually or using CLI.

PS if there is a way to change the mobile device as well, it would be really helpful for my web testing.

Thanks!

**Comment by adamraine** [comment]
The network speed, CPU power, and emulated device can be adjusted in the [Lighthouse config](https://github.com/GoogleChrome/lighthouse/blob/main/docs/configuration.md).

> I would like to modify the Network speed and CPU processing power for Mobile and Web Lighthouse Performance testing. Please let me know if there is a way to do it manually or using CLI.

See also: https://github.com/GoogleChrome/lighthouse/blob/main/docs/throttling.md

> PS if there is a way to change the mobile device as well, it would be really helpful for my web testing.

See also: https://github.com/GoogleChrome/lighthouse/blob/main/docs/emulation.md
